packages/ipa/src/service/show.coffee.md

```diff
@@ -0,0 +1,76 @@
+
+# `nikita.ipa.service.show`
+
+Retrieve service information from FreeIPA.
+
+## Options
+
+* `principal` (string, required)   
+  Name of the service to add.
+
+## Exemple
+
+```js
+require("nikita")
+.ipa.service.show({
+  principal: "myprincipal/my.domain.com",
+  connection: {
+    url: "https://ipa.domain.com/ipa/session/json",
+    principal: "admin@DOMAIN.COM",
+    password: "mysecret"
+  }
+}, function(err, {result}){
+  console.info(err ? err.message :
+    `Service is ${result.principal[0]}`)
+  // If service is missing, `err` looks like:
+  // { code: 4001
+  // message: 'missing: service not found' }
+  // If service exists, `result` looks like:
+  // { dn: 'krbprincipalname=myservice/freeipa.nikita.local@NIKITA.LOCAL,cn=services,cn=accounts,dc=nikita,dc=local',
+  //  krbprincipalname: [Array],
+  //  has_keytab: false,
+  //  managedby_host: [Array],
+  //  krbcanonicalname: [Array] }
```

Comment:
You can illustrate this with a code example instead of a commit, eg

```js
switch(err.code){
  case 4001:
    assert("missing: service not found", err.message)
  break
}
```

Reply:
you can update the sample in "service/show" and squash your commit into one before I merge. Thank you